runtime/autoload/man.vim

```diff
@@ -12,6 +12,11 @@ catch /E145:/
   " Ignore the error in restricted mode
 endtry
 
+try
```

Comment:
It was a test. I didn't have a way to verify how linux does it but found an online shell interpreter.